shared/gh/js/views/tenant-admin/gh.admin-edit-organiser.js

```diff
@@ -210,32 +210,31 @@ define(['gh.core', 'jquery.jeditable'], function(gh) {
              * @private
              */
             var closeAutoSuggest = function(ev) {
-                // Only close the input if the focus was lost on an element outside of the organiser container
-                if (!$(ev.target).closest('.gh-event-organisers').length || !$(ev.relatedTarget).closest('.gh-event-organisers').length) {
-                    // The form shouldn't be submitted when a click was triggered by pressing the space
-                    if (ev.type !== 'click') {
-                        // Submit the AutoSuggest field
-                        submitAutoSuggest(this, original);
-
-                        // Calculate how long it takes the user to change the date
-                        timeFromStart = (new Date() - timeFromStart) / 1000;
-                        // Track the user editing organisers
-                        gh.utils.trackEvent(['Data', 'Lecturer edit', 'Completed'], {
-                            'time_from_start': timeFromStart
-                        });
-
-                        // If there is a relatedTarget set on the event that triggered the function, focus on it
-                        if ($(ev.relatedTarget).length) {
-                            $(ev.relatedTarget).focus();
-                        }
-                    }
-                }
+                // Submit the AutoSuggest field
+                submitAutoSuggest(this, original);
+
+                // Track the user editing organisers
+                timeFromStart = (new Date() - timeFromStart) / 1000;
+                gh.utils.trackEvent(['Data', 'Lecturer edit', 'Completed'], {
+                    'time_from_start': timeFromStart
+                });
             };
 
```

Comment:
Can we removing threedotting from the list items in the admin UI? Isn't it much more usable when the full list of lecturers shows?